src/vcSettings.h

```diff
@@ -95,6 +97,21 @@ enum vcTileRendererFlags
   vcTRF_OnlyRequestVisibleTiles = 0x1,
 };
 
+enum vcImageFormats
+{
+  vcIF_PNG,
+  vcIF_BMP,
+  vcIF_TGA,
+  vcIF_JPG
```

Comment:
Just because we _can_ offer these doesn't mean we should. Just having PNG and JPG is enough.